examples/visual-tests/amp-story/amp-story-bot-rendering.html

```diff
@@ -5,11 +5,10 @@
     <script async src="https://cdn.ampproject.org/v0.js"></script>
     <script async custom-element="amp-story" src="https://cdn.ampproject.org/v0/amp-story-1.0.js"></script>
     <script async custom-element="amp-social-share" src="https://cdn.ampproject.org/v0/amp-social-share-0.1.js"></script>
-    <link href='https://fonts.googleapis.com/css?family=Georgia|Open+Sans|Roboto' rel='stylesheet' type='text/css'>
-    <link href="https://fonts.googleapis.com/css?family=Roboto+Condensed" rel="stylesheet">
```

Comment:
Just to confirm, these appear to have nothing to do with the document and are just dead code? 
We're not testing anything with bot rendering by importing these?

Reply:
Yep, they're just dead code and just add latency.